Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -26,6 +26,12 @@ public extension StorageType {
         return firstObject(ofType: Site.self, matching: predicate)
     }
 
+    /// Retrieves all stored sites.
+    ///
+    func loadAllSites() -> [Site]? {
```

Comment:
I don't think this has to be optional. Unless I'm missing something? 🤪 

```suggestion
    func loadAllSites() -> [Site] {
```

Reply:
nice catch! I was following another similar function that returns an optional list, but it isn't necessary in this case. updated in https://github.com/woocommerce/woocommerce-ios/pull/5241/commits/3194c17ae5c23de8471d358e7e66b9b0c3e101e4